packages/app/src/cli/services/deploy.ts

```diff
@@ -184,9 +177,23 @@ async function configFor(extension: UIExtension, app: AppInterface) {
     case 'pos_ui_extension':
     case 'product_subscription': {
       const result = await getUIExtensionRendererVersion(type, app)
-      if (result === 'not_found') throw RendererNotFoundBug(type)
+      if (result === 'not_found') {
```

Comment:
Why do we need to change this existing code?

Reply:
This was caught by a lint error that forbids factory functions for errors ([code here](https://github.com/Shopify/cli/blob/main/eslint-rules/no-error-factory-functions.cjs))

Because I edited the file eslint complained and I had to update the code, even though I hadn't touched it.